raiden/tests/unit/transfer/mediated_transfer/test_initiator.py

```diff
@@ -127,9 +130,9 @@ def test_next_route():
     assert state.route == routes[0], 'a initialized state must use the first valid route'
 
     assert state.routes.available_routes == routes[1:]
-    assert len(state.routes.ignored_routes) == 0
-    assert len(state.routes.refunded_routes) == 0
-    assert len(state.routes.canceled_routes) == 0
+    assert not state.routes.ignored_routes
```

Comment:
if you want some sort of 'type safety' the best way is to use `isinstance`, the ignored_routes could also be a set or a dict and `len` would work. In general the preferred approach is the most readable one, and for checking if a collection has elements `assert collection` is the preferred way.